lapce-data/src/explorer.rs

```diff
@@ -38,12 +39,25 @@ pub enum Naming {
         /// The folder that the file/directory is being created within
         base_path: PathBuf,
     },
+    /// Duplicating an existing file
+    Duplicating {
+        /// The index that the file being created should appear at
+        /// Note that when duplicating, it is not yet actually created.
+        list_index: usize,
+        /// Indentation level
+        indent_level: usize,
+        /// The folder that the file/directory is being created within
+        base_path: PathBuf,
+        /// The name of the file being duplicated
+        name: OsString,
```

Comment:
This should be a `String` rather than `OsString`, we don't operate on raw platform strings (at least not yet)